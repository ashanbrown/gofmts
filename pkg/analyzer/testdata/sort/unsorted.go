package sort

//gofmts:sort
const Z = 1 // want "block is unsorted"
const A = 2

const (
	//gofmts:sort
	// ignore this
	Z1 = 1 // want "block is unsorted"
	A2 = 2
)

const (
	//gofmts:sort
	Z3 = 1 // want "block is unsorted"
	// move this
	A3 = 2
)
