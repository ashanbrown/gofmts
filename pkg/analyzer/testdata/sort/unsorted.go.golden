package sort

//gofmts:sort
const A = 2
const Z = 1 // want "block is unsorted"

const (
	//gofmts:sort
	// ignore this
	A2 = 2
	Z1 = 1 // want "block is unsorted"
)

const (
	//gofmts:sort
	// move this
	A3 = 2
	Z3 = 1 // want "block is unsorted"
)
